main/src/cgeo/geocaching/settings/Settings.java

```diff
@@ -611,6 +612,10 @@ public static boolean isRegisteredForGeokrety() {
         return getGeokretySecId() != null;
     }
 
+    public static boolean get24HourStyle() {
```

Comment:
A method named like that should not return a boolean. Or, rather, should be renamed, into something like `use24HoursStyle`.

Anyway, I think it should go away, see below.